### PR TITLE
Mention tools in README

### PR DESCRIPTION
**Pull request by XVilka**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [x] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Very few people are aware that Rizin also provides different helpful tools apart from `rizin` itself. Put them on the "first page," therefore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,6 +87,21 @@ ZIMG, MBN/SBL bootloader, ELF coredump, MDMP (Windows minidump), DMP (Windows pa
 WASM (WebAssembly binary), Commodore VICE emulator, QNX,
 Game Boy (Advance), Nintendo DS ROMs and Nintendo 3DS FIRMs.
 
+## Tools
+
+Apart from the main tool - `rizin` there are also other tools tailored for specific purpose and
```
Comment (Rot127) [inline README.md]:
```suggestion
Apart from the main tool `rizin`, there are also other tools tailored for specific purposes and 
```

File: README.md [hunk 2]
```diff
@@ -87,6 +87,21 @@ ZIMG, MBN/SBL bootloader, ELF coredump, MDMP (Windows minidump), DMP (Windows pa
 WASM (WebAssembly binary), Commodore VICE emulator, QNX,
 Game Boy (Advance), Nintendo DS ROMs and Nintendo 3DS FIRMs.
 
+## Tools
+
+Apart from the main tool - `rizin` there are also other tools tailored for specific purpose and 
+useful for shell scripting or as a separate standalone tools:
```
Comment (Rot127) [inline README.md]:
```suggestion
useful for shell scripting or as separate standalone tools:
```

File: README.md [hunk 3]
```diff
@@ -87,6 +87,21 @@ ZIMG, MBN/SBL bootloader, ELF coredump, MDMP (Windows minidump), DMP (Windows pa
...
+
+- `rz-bin` - provides all kind of information about binary formats
+- `rz-asm` - a command-line assembler and disassemblers
+- `rz-diff` - a tool to compare two binaries as raw data or analyzed executables
+- `rz-hash` - allows to calculate different hashes or even encrypt the Data
```
Comment (Rot127) [inline README.md]:
```suggestion
- `rz-hash` - allows to calculate different hashes or even encrypt data
```

**Review by Rot127 — CHANGES_REQUESTED** [review]
Definitely a good addition IMHO.

File: README.md [hunk 4]
```diff
@@ -87,6 +87,21 @@ ZIMG, MBN/SBL bootloader, ELF coredump, MDMP (Windows minidump), DMP (Windows pa
...
+Apart from the main tool `rizin`, there are also other tools tailored for specific purposes and 
+useful for shell scripting or as separate standalone tools:
+
+- `rz-bin` - provides all kind of information about binary formats
+- `rz-asm` - a command-line assembler and disassemblers
+- `rz-diff` - a tool to compare two binaries as raw data or analyzed executables
+- `rz-hash` - allows to calculate different hashes or even encrypt data
+- `rz-gg` - a small "eggs" code generator useful for exploitation purposes
+- `rz-find` - binary analog of `find` tool, allowing to search patterns and bit masks
+- `rz-sign` - tool to work with FLIRT signatures
```
Comment (wargio) [inline README.md]:
```suggestion
- `rz-sign` - tool to create, convert and parse FLIRT signatures
```

**Review by wargio — APPROVED** [review]

**Pull request merged**